cli/cmd/interact/match.go

```diff
@@ -0,0 +1,35 @@
+package interact
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// MatchingStringInput waits for user to input a string that matches expr.
+// If lowercase is true than before matching string is transformed to lowercase.
+func MatchingStringInput(expr string, lowercase bool, question string) (string, error) {
```

Comment:
Wouldn't it make more sense to call the flag `caseInsensitive` (or `ignoreCase` or similar) rather than `lowercase`? We care about the matching being case insensitive, not how this is implemented internally.

Reply:
Not only does it match case Insensitive (or ignore Case) but also modifies the input to be lowercased.